src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandMap.js

```diff
@@ -54,67 +20,191 @@ const UNICODE_COMMANDS = {
  * @final
  */
 export const AIRCRAFT_COMMAND_MAP = {
```

Comment:
nice work here!  I really like the simplification you've done here.

Reply:
🎉